packages/cli/tests/images/build.js

```diff
@@ -91,9 +91,8 @@ exports.preload.head = `
 	<link rel=\\"manifest\\" href=\\"\\/manifest\\.json\\">
 	<link rel=\\"preload\\" href=\\"\\/bundle\\.\\w{5}\\.js\\" as=\\"script\\">
 	<link rel=\\"preload\\" href=\\"\\/route-home\\.chunk\\.\\w{5}\\.js\\" as=\\"script\\">
-	<link rel=\\"preload\\" href=\\"\\/2\\.chunk\\.\\w{5}\\.css\\" as=\\"style\\">
 	<link rel=\\"preload\\" href=\\"\\/route-home\\~route-route66\\~route-route89\\.chunk\\.\\w{5}\\.js\\" as=\\"script\\">
-	<link rel=\\"preload\\" href=\\"\\/1\\.chunk\\.\\w{5}\\.css\\" as=\\"style\\">
+	<link rel=\\"preload\\" href=\\"\\/route-home\\.chunk\\.\\w{5}\\.css\\" as=\\"style\\">
```

Comment:
I am concerned that one of the css chunk preloads needs to be removed rather than adjusted here. I expected there to be a line that looks like
```javascript
<link rel=\\"preload\\" href=\\"\\/route-home\\~route-route66\\~route-route89\\.chunk\\.\\w{5}\\.css\\" as=\\"style\\">
```

Reply:
i don't understand the `~route-route66\\~route-route89` part?
so for this page, the home css and the route css should be loaded... why multiple route name in the same tag?

Reply:
perhaps i misunderstand the question. isn't that some new code splitting introduced on the `next` branch? in my local project the name represents the routes that share the code - all those names are joined by `~`.

Reply:
hmm - this is strange. Unless contenthash is producing slashes, it seems like the URL format here should still just be `/xx.chunk.12345.css`:

https://github.com/developit/preact-cli/blob/next/packages/cli/lib/lib/webpack/webpack-base-config.js#L203-L207

Reply:
the code splitting happns only under some conditions and i see that 30kb condition wont match here.. so the above code looks fine

Reply:
so we can leave as is?

Reply:
I Guess so